NEWS.md

```diff
@@ -65,6 +65,9 @@ added for subjects who have both an event or censoring and an observation in
 `HYPERURICEMIA` as grade criteria based on `ANRHI` only.  This metadata holds criteria for lab grading
 based on [Common Terminology Criteria for Adverse Events (CTCAE) v5.0](https://ctep.cancer.gov/protocoldevelopment/electronic_applications/ctc.htm) (#1650)
 
+- The following functions are being deprecated (#1683)
+  - `derive_vars_disposition_reason()`
+
```

Comment:
This should go in Breaking changes and not in Documentation.  

Actually, I think something happened with merge s the Documentation has the New vignette "Creating a PK..." but then goes into breaking changes.    So that needs to get shuffled around.  Is it okay in `devel`?